rpi-update

```diff
@@ -281,6 +310,19 @@ function noobs_fix {
 
 }
 
+function get_hash_date {
+	commit_api="https://api.github.com/repos/Hexxeh/rpi-firmware/git/commits/"
```

Comment:
I think for style-consistency this should be upper-cased to `COMMIT_API`, and it should be dynamically calculated from `$REPO_URI` in the same way that `REPO_API` is on line 364; in case the user supplies a custom `REPO_URI`.

In fact, perhaps it makes sense to have `REPO_API=${REPO_URI/github.com/api.github.com\/repos}` up at the top of the script, and then line 315 could use `$REPO_API/git/commits/$1` and line 365 etc. could use `$REPO_API/git/refs/heads/${BRANCH}`

Reply:
I've made this change in #279 :slightly_smiling_face: